### DISCUSSION TITLE
Popup "You are connected with your account" already logged

### DISCUSSION THREAD

**Discussion by ruben-certhis** [Web3Auth/Web3Auth#802] [Help]
hi , i have problem , when the user is already logged and i use

`await web3auth.configureAdapter(openloginAdapter);`

it shows the popup "You are connected with your account"  when the user refresh the page.

without 
`await web3auth.configureAdapter(openloginAdapter);`

 it works normaly  and doesn't  show the popup "You are connected with your account" when the user refresh the page.

how can i use openloginAdapter and not having troubles with the popups?

(i need to use the  openloginAdapter for custom whitelist domain)

**Comment by shahbaz17** [comment]
Hey @ruben-certhis 

Please share your log and code snippet.
